src/main/webapp/css/style.css

```diff
@@ -1,5 +1,5 @@
 .favourite {
```

Comment:
The style guide actually seems to mention 2 space indentation as the norm. https://google.github.io/styleguide/htmlcssguide.html#Indentation.

Is there a reason you changed to 4 spaces?

Reply:
There is no reason I was just trying to make it consistent with Timothy's code